glean-core/tests/labeled.rs

```diff
@@ -251,3 +252,70 @@ fn dynamic_labels_regex_mimsatch() {
         snapshot
     );
 }
+
+#[test]
+fn seen_labels_get_reloaded_from_disk() {
```

Comment:
Will this allow us to enable [this test](https://github.com/mozilla/glean/blob/93736f8598b607f77c96bf9cda266c7716dde987/glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt#L436) again?

Reply:
once I understand the mocking part

Reply:
I can help you with that, no worries :) ping me about i!

Reply:
I took a look and can't really see how to re-enable that tests (or even that we _should_ re-enable it):

* It's mocking sharedpreferences, which we are not using anymore
* We can't easily inject data into the store, without going through the rudst part anyway.
* This case is covered by the Rust test

How about removing that kotlin test case?
wdyt, @Dexterp37?

Reply:
I think you're right. Let's remove the test and add something like this in its place:

```
// SKIPPED `test seen labels get reloaded from disk`
// REASON XYZ HERE
```

(here or in a follow-up)